features/unit/sourcemap.feature

```diff
@@ -0,0 +1,35 @@
+@unit.sourcemap
+Feature: SourceMap
+
+    @unit.sourcemap.full
```

Comment:
@barnjamin Are the following subtags needed?  
* `@unit.sourcemap.full`
* `@unit.sourcemap.pctoline`
* `@unit.sourcemap.linetopc`

The downstream SDKs rely on `@unit.sourcemap`.  Rationale for removing:  Presents fewer chances for downstream SDKs to be out-of-sync (e.g. miss running a test).

It's possible I'm missing the rationale for retaining them.  Feel welcomed to suggest retaining the tags.

Reply:
You're right, I've removed all but the top tag